app/views/user_sessions/new.html.erb

```diff
@@ -40,8 +34,5 @@
 </div>
 
 <p>
-  <%= ts("Forgot your password or user name?") %> <%= link_to ts("Reset password"), new_password_path %>
-  <% if @admin_settings.account_creation_enabled? %>
-    <br \><%= ts("Don't have an account?") %> <%= link_to ts('Create an account now.'), new_user_path %>
-  <% end %>
+  <%= ts("Forgot your password or user name?") %> <%= link_to ts("Reset password"), new_password_path %> or <%= new_account_link %>
```

Comment:
I think we might run into some weirdness with this part: `or <%= new_account_link %>`. If account creation is disabled, this sentence will end up saying, "Forgot your password or user name? Reset password or Account creation disabled."